doc/changelog.md

```diff
@@ -50,6 +50,11 @@
 
 <h3>Improvements</h3>
 
+* Added support for IsingZZ gate in Catalyst frontend. 
+Previously, the IsingZZ gate would be decomposed into 
+a CNOT and RZ gates. However, this is not needed as 
+the PennyLane-Lightning simulator supports this gate.
+
```

Comment:
Don't forget to add your name below :) I think this PR is good and we will likely merge it :)

Some small suggestions for the commit changelog message.

```suggestion
* Added support for IsingZZ gate in Catalyst frontend. 
  Previously, the IsingZZ gate would be decomposed into 
  a CNOT and RZ gates. However, this is not needed as 
  the PennyLane-Lightning simulator supports this gate.
  [(#730)](https://github.com/PennyLaneAI/catalyst/pull/730)
```